docs/spark/spark-ddl.md

```diff
@@ -91,11 +91,15 @@ Supported transformations are:
 
 ## `CREATE TABLE ... AS SELECT`
 
-Iceberg supports CTAS as an atomic operation when using a [`SparkCatalog`](../spark-configuration#catalog-configuration). CTAS is supported, but is not atomic when using [`SparkSessionCatalog`](../spark-configuration#replacing-the-session-catalog).
+Iceberg supports CTAS as an atomic operation when using a [`SparkCatalog`](../spark-configuration#catalog-configuration). CTAS is supported, but is not atomic when using [`SparkSessionCatalog`](../spark-configuration#replacing-the-session-catalog). 
+
+The newly created table won't inherit the partition spec and table properties from the source table in SELECT, you can use PARTITIONED BY and TBLPROPERTIES  in CTAS to declare partition spec and table properties for the new table.
 
 ```sql
 CREATE TABLE prod.db.sample
 USING iceberg
+PARTITIONED BY (part)
+TBLPROPERTIES ('key'='value')
```

Comment:
Documentation should start with simple examples and add more complexity if needed.

In this case, I think it is valid to add a `PARTITIONED BY` example, but not by modifying the first simple example. Please add the new paragraph and a separate example after the existing introduction to CTAS.